enos/modules/vault_cluster/templates/install-packages.sh

```diff
@@ -33,16 +33,29 @@ function retry {
 }
 
 echo "Installing Dependencies: $packages"
-if [ -f /etc/debian_version ]; then
-  # Do our best to make sure that we don't race with cloud-init. Wait a reasonable time until we
-  # see ec2 in the sources list. Very rarely cloud-init will take longer than we wait. In that case
-  # we'll just install our packages.
-  retry 7 grep ec2 /etc/apt/sources.list || true
-
-  cd /tmp
-  retry 5 sudo apt update
-  retry 5 sudo apt install -y "$${packages[@]}"
-else
-  cd /tmp
-  retry 7 sudo yum -y install "$${packages[@]}"
-fi
+  # If this distro comes with yum or apt, use that to install packages
```

Comment:
I think this whole thing should be refactored to a switch case of some kind but wasn't able to get to it today.